yoti-sdk-api/src/main/java/com/yoti/api/client/identity/NotificationRepresentation.java

```diff
@@ -0,0 +1,109 @@
+package com.yoti.api.client.identity;
+
+import static com.yoti.api.client.spi.remote.util.Validation.notNullOrEmpty;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public final class NotificationRepresentation {
+
+    @JsonProperty(Property.URL)
+    private final String url;
+
+    @JsonProperty(Property.METHOD)
+    private final String method;
+
+    @JsonProperty(Property.VERIFY_TLS)
+    private final boolean verifyTls;
+
+    @JsonProperty(Property.HEADERS)
+    private final Map<String, String> headers;
+
+    NotificationRepresentation(String url, String method, boolean verifyTls, Map<String, String> headers) {
+        notNullOrEmpty(url, Property.URL);
+
+        this.url = url;
+        this.method = method;
+        this.verifyTls = verifyTls;
+        this.headers = headers;
+    }
+
+    public String getUrl() {
+        return url;
+    }
+
+    public String getMethod() {
+        return method;
+    }
+
+    public boolean isVerifyTls() {
+        return verifyTls;
+    }
+
+    public Map<String, String> getHeaders() {
+        return headers;
+    }
+
+    public Builder builder() {
+        return new Builder();
+    }
```

Comment:
Maybe make it so this requires the URI? Out of all the properties it's the only thing that is actually "required" (everything else has defaults). It would be one less builder setter method (slightly less verbose).